libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -1521,7 +1521,7 @@ void CompilerUtils::copyContractCodeToMemory(ContractDefinition const& contract,
 				_context.compiledContract(contract) :
 				_context.compiledContractRuntime(contract);
 			// pushes size
-			auto subroutine = _context.addSubroutine(assembly);
+			auto subroutine = _context.addSubroutine(assembly, _creation);
```

Comment:
Or put differently: I'm not sure if we need this change - I'd expect weird thing to happen without it, but couldn't come up with a case.